editor/script/find_in_files.cpp

```diff
@@ -724,6 +726,12 @@ FindInFilesPanel::FindInFilesPanel() {
 		_status_label->set_focus_mode(FOCUS_ACCESSIBILITY);
 		hbc->add_child(_status_label);
 
+		_keep_results_button = memnew(CheckButton);
+		_keep_results_button->set_text(TTRC("Keep Results"));
+		_keep_results_button->set_tooltip_text(TTRC("Keep these results and show subsequent results in a new window"));
+		_keep_results_button->set_pressed(true);
```

Comment:
I think this should be set to false by default. Most of the time, you don't need to keep the results.

I was considering if it should be saved as well, but since there is one per tab it that would have issues.